app.js

```diff
@@ -31,6 +31,31 @@ function initApp(options) {
     if(!app.conf.port) { app.conf.port = 8888; }
     if(!app.conf.interface) { app.conf.interface = '0.0.0.0'; }
     if(!app.conf.compression_level) { app.conf.compression_level = 3; }
+    if(app.conf.cors === undefined) { app.conf.cors = '*'; }
+    if(!app.conf.csp) {
+        app.conf.csp =
+            "default-src 'self'; object-src 'none'; media-src *; img-src *; style-src *; frame-ancestors 'self'";
+    }
+
+    // set outgoing proxy
+    if(app.conf.proxy) {
```

Comment:
Other protocols can be proxied as well, in which case just 'proxy' would be confusing. I agree that picking one is better than requiring both. Maybe `http_https_proxy`?